config/rbac/applicationsnapshot_role_binding.yaml

```diff
@@ -1,12 +0,0 @@
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: applicationsnapshot-role-binding
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: applicationsnapshot-editor-role
-subjects:
-  - kind: ServiceAccount
-    name: controller-manager
-    namespace: system
```

Comment:
(optional)
I'm just curious, please feel free to resolve this.
I see `applicationsnapshot_role_binding.yaml` and `applicatoin_role_binding.yaml` were similar things, and this one was selected for removal. Were these files redundant before, or just now after the change?

Reply:
Git made it hard to follow, but what we really had was `application_role_binding.yaml` and `applicatoin_role_binding.yaml`. I removed the `applicatoin` one and renamed `applicationsnapshot_role_binding` to `snapshot_role_binding` but git displays it differently. The `applicatoin_role_binding` was not part of the kustomization file, so I am not sure why it was there since it wasn't doing anything (hence we I have one file being removed here)